quickjs.c

```diff
@@ -1617,7 +1617,7 @@ JSRuntime *JS_NewRuntime2(const JSMallocFunctions *mf, void *opaque)
 
     memset(&ms, 0, sizeof(ms));
     ms.opaque = opaque;
-    ms.malloc_limit = -1;
+    ms.malloc_limit = 0;
```

Comment:
Not sure if the original intentio here was to set it to SIZE_MAX, but it feels kinda wrong.